h2o-docs/src/booklets/v2_2015/source/sw/sections/pysparkling.tex

```diff
@@ -72,20 +72,20 @@ \subsection{Getting Started:}
 Then create an H2O cloud inside the Spark cluster and import H2O-Python package:
 \begin{lstlisting}[style=Scala]
 from pysparkling import *
-hc= H2OContext(sc).start()
+hc = H2OContext(spark).start()
 import h2o
 \end{lstlisting}
 
 Or to launch as a Spark Package application:
 \begin{lstlisting}[style=Bash]
-$SPARK_HOME/bin/spark-submit --packages ai.h2o:sparkling-water-core_2.11:2.1.8 --py-files $SPARKLING_HOME/py/dist/pySparkling-2.1.8-py2.7.egg
+$SPARK_HOME/bin/spark-submit --packages ai.h2o:sparkling-water-core_2.11:2.2.2 --py-files $SPARKLING_HOME/py/dist/pySparkling-2.2.2-py2.7.egg
```

Comment:
The path should be `$SPARKLING_HOME/py/build/dist/h2o_pysparkling_2.2-2.2.2.zip` 

We now use zip files to package pysparkling instead of eggs.